presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotStreamingQueryClient.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.pinot;
+
+import org.apache.pinot.common.config.GrpcConfig;
+import org.apache.pinot.common.proto.Server;
+import org.apache.pinot.common.utils.grpc.GrpcQueryClient;
+import org.apache.pinot.common.utils.grpc.GrpcRequestBuilder;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+/**
+ * Grpc based Pinot query client.
+ */
+public class PinotStreamingQueryClient
```

Comment:
might be worth separating the GRPC based Pinot client into a separate commit and the pom upgrades into a separate commit

Reply:
done.

Reply:
@xiangfu0 Did you push the commits. I am still seeing only 1 commit in this PR. :)

Reply:
I force pushed it.

Reply:
can you please double check the commits tab in this PR. still seeing only 1 commit.
> [Upgrade pinot to 0.12.1](https://github.com/prestodb/presto/pull/19992/commits/4981645e67091198d490fccb8d2d299978aa7e44)

Reply:
Yes, it suppose to be one commit. I amended the previous commit then force pushed it.

Reply:
Sorry if you misunderstood my review comment. My suggestion was to split into two different commits for clarity.

Reply:
Got it, added another PR to  simplify the dependencies. Thanks for your clarification.

Reply:
Can you help rerun the canceled test?

Reply:
done